app/models/course/assessment/assessment_ability.rb

```diff
@@ -157,4 +158,8 @@ def allow_students_read_own_assessment_answers
   def allow_staff_read_assessment_answers
     can :read, Course::Assessment::Answer, discussion_topic: course_staff_hash
   end
+
+  def allow_students_read_submission_question
+    can :read, Course::Assessment::SubmissionQuestion
```

Comment:
they can only read their on submission_question, if declare like this then it's all submission questions.